SofaKernel/modules/SofaBaseTopology/MeshTopology.cpp

```diff
@@ -1589,6 +1589,19 @@ const MeshTopology::HexahedraAroundQuad& MeshTopology::getHexahedraAroundQuad(Qu
 
 
 
+const vector< MeshTopology::EdgesAroundVertex >& MeshTopology::getEdgesAroundVertexArray()
+{
+    if (m_edgesAroundVertex.empty())	// this method should only be called when the array exists.
+    {
+#ifndef NDEBUG
+        sout << "Warning. [MeshTopology::getEdgesAroundVertexArray] EdgesAroundVertex array is empty." << sendl;
+#endif
```

Comment:
`msg_warning() << "getEdgesAroundVertexArray: EdgesAroundVertex array is empty.";`

What is the purpose of NDEBUG?